### DISCUSSION TITLE
Cloning a row

### DISCUSSION THREAD

**Discussion by d41ly** [nocodb/nocodb#7411] [General]
According to this https://github.com/nocodb/nocodb/issues/435 and this https://github.com/nocodb/nocodb/pull/5035 the functionality to clone rows was implemented more than a year ago, but for the life of me I cannot find it in the interface. Was it lost after the recent revamp? Request it to be readded.

**Comment by dstala** [comment]
@nocodbenjoyer Duplicate record functionality was added to expanded record only. We had not introduced it onto the Grid yet

<img width="1201" alt="Screenshot 2024-01-13 at 8 31 53 AM" src="https://github.com/nocodb/nocodb/assets/86527202/4a46ff6c-6344-440c-b436-45a23b14146c">
